### PR TITLE
Add sample reddit checker example

### PR DESCRIPTION
**Pull request by pchng**
Adds a script that allows you to search a subreddit for posts that match all the supplied terms.

### CODE REVIEW

File: samples/reddit/command.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  if (numFound == 0) {
+    message += '\nNo results found.'
+  }
+
+  await ctx.client.sendEphemeral(message);
+};
+
+const help = async (ctx) => {
+  await ctx.client.sendEphemeral(`This command searches a specified subreddit for new posts with titles containing the supplied terms. Usage:
```
Comment (matthewzhaocc) [inline samples/reddit/command.js]: This should be standardized to be a ctx.client.send.

Comment (pchng) [reply samples/reddit/command.js]: I'll change both instances to `ctx.client.send`, on lines 63 and 67 here.

**Review by matthewzhaocc — APPROVED** [review]
Thanks for building this example! 😄

**Comment by pchng** [comment]
> Thanks for building this example! 😄

👍

**Pull request merged**